readme.md

```diff
@@ -8,28 +8,46 @@
 
 ## Overview
 
+Chief-of-state (COS) is a dockerized persistence framework for building modern, clustered, reactive applications.
+COS encourages best practices in CQRS and event-sourcing through simple, language-agnostic interfaces via gRPC, and it
+allows developers to describe their schema with Protobuf. Under the hood, COS leverages Akka to scale out and guarantee
+performant, reliable persistence.
+
+Chief-of-state was built by Namely with the following principles:
+* Wire format should be the same as persistence (Protobuf is a better schema definition than DDL)
```

Comment:
Is protobuf really better, or does it serve different purposes? I would consider striking the parenthetical.

Reply:
i'm trying to accomplish too much here, and i can see counter arguments to this bullet that weren't my intent. I'll rethink this.